Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -341,7 +342,7 @@ class SourceGenerator {
     }
 
     /// creates all the source files and groups they belong to for a given targetSource
-    private func getGroupSources(targetType: PBXProductType, targetSource: TargetSource, path: Path, isBaseGroup: Bool)
+    private func getGroupSources(targetType: PBXProductType, targetSource: TargetSource, path: Path, createIntermediateGroups: Bool, hasCustomParent: Bool, isBaseGroup: Bool)
```

Comment:
Do we need to add these params? The properties live on `targetSource`

Reply:
I will check